cargo-process.el

```diff
@@ -145,7 +150,12 @@
 (defun cargo-process--start (name command)
   "Start the Cargo process NAME with the cargo command COMMAND."
   (let ((buffer (concat "*Cargo " name "*"))
-        (command (cargo-process--maybe-read-command command))
+        (command (cargo-process--maybe-read-command
+                  (if (string= cargo-process--custom-path-to-bin "")
```

Comment:
A couple of things to maybe clean this up:

Default should be `nil`
And maybe to clean this up a bit, something like this?
````
 (let* ((buffer (concat "*Cargo " name "*"))
        (path cargo-process--custom-path-to-bin)
        (path (and path (file-name-as-directory path)))
        (command (cargo-process--maybe-read-command (concat path command)))
        (project-root (cargo-process--project-root)))
````
Love it or hate it, it's my preference :)
If Elisp had Clojure's threading macro it would've been much nicer..